libpng not downloadable from sourceforge

```
2020-06-03 20:26:27+00:00 WGET https://download.sourceforge.net/libpng/libpng-1.6.34.tar.gz

if ! wget -O "/root/project/packages/libpng-1.6.34.tar.gz.tmp" https://download.sourceforge.net/libpng/libpng-1.6.34.tar.gz ; then exit 1 ; fi ; mv "/root/project/packages/libpng-1.6.34.tar.gz.tmp" "/root/project/packages/libpng-1.6.34.tar.gz" 

--2020-06-03 20:26:27--  https://download.sourceforge.net/libpng/libpng-1.6.34.tar.gz

Resolving download.sourceforge.net (download.sourceforge.net)... 216.105.38.13

Connecting to download.sourceforge.net (download.sourceforge.net)|216.105.38.13|:443... 
connected.

HTTP request sent, awaiting response... 301 Moved Permanently

Location: https://downloads.sourceforge.net/project/libpng/libpng16/1.6.34/libpng-1.6.34.tar.gz [following]

--2020-06-03 20:26:27--  https://downloads.sourceforge.net/project/libpng/libpng16/1.6.34/libpng-1.6.34.tar.gz

Resolving downloads.sourceforge.net (downloads.sourceforge.net)... 216.105.38.13

Connecting to downloads.sourceforge.net (downloads.sourceforge.net)|216.105.38.13|:443... connected.

HTTP request sent, awaiting response... 
302 Found

Location: https://managedway.dl.sourceforge.net/project/libpng/libpng16/1.6.34/libpng-1.6.34.tar.gz [following]

--2020-06-03 20:26:27--  https://managedway.dl.sourceforge.net/project/libpng/libpng16/1.6.34/libpng-1.6.34.tar.gz

Resolving managedway.dl.sourceforge.net (managedway.dl.sourceforge.net)... 208.79.213.6

Connecting to managedway.dl.sourceforge.net (managedway.dl.sourceforge.net)|208.79.213.6|:443... 
connected.

GnuTLS: Error in the pull function.

Unable to establish SSL connection.

make: *** [Makefile:382: /root/project/packages/libpng-1.6.34.tar.gz] Error 1

```

--- a/modules/libpng
+++ b/modules/libpng
@@ -3,7 +3,7 @@ modules-$(CONFIG_CAIRO) += libpng
 libpng_version := 1.6.34
 libpng_dir := libpng-$(libpng_version)
 libpng_tar := libpng-$(libpng_version).tar.gz
-libpng_url := https://download.sourceforge.net/libpng/$(libpng_tar)
+libpng_url := https://github.com/glennrp/libpng-releases/raw/master/$(libpng_tar)
 libpng_hash := 574623a4901a9969080ab4a2df9437026c8a87150dfd5c235e28c94b212964a7
 
 libpng_configure := \

